pkgs/development/compilers/elm/default.nix

```diff
@@ -27,9 +27,23 @@ let
             The elm-format expression is updated via a script in the https://github.com/avh4/elm-format repo:
             `package/nix/build.sh`
             */
-            #elm-format = justStaticExecutables (doJailbreak (self.callPackage ./packages/elm-format.nix {}));
+            elm-format = justStaticExecutables (overrideCabal (self.callPackage ./packages/elm-format.nix {}) (drv: {
+              # GHC 8.8.1 support
+              # https://github.com/avh4/elm-format/pull/640
+              patches = [(
+                fetchpatch {
+                  url = "https://github.com/turboMaCk/elm-format/commit/4f4abdc7117ed6ce3335f6cf39b6495b48067b7c.patch";
+                  sha256 = "052s9shxz92p25xgw8rmgzbhwxq9b8jdp16ng563sb3h00m64fm0";
```

Comment:
@domenkozar sorry for stupid question but what are you using to prefetch patches like this? I think this hash can be wrong as on my other machine I it says this should be `1zqk6q6w0ph12mnwffgwzf4h1hcgqg0v09ws9q2g5bg2riq4rvd9`.

Reply:
I just change the hash to be wrong and `nix-build -A elmPackages.elm-format`

Reply:
makes sense, thanks. I've used `nix-prefetch-url` on a desktop. not sure why that worked there for some. I'll know better next time.